[2022-04-24 01:52:59,144] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.crash_bigquery_external_table_task manual__2022-04-24T01:52:31.743537+00:00 [queued]>
[2022-04-24 01:52:59,214] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.crash_bigquery_external_table_task manual__2022-04-24T01:52:31.743537+00:00 [queued]>
[2022-04-24 01:52:59,216] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 01:52:59,218] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-04-24 01:52:59,219] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 01:52:59,279] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): crash_bigquery_external_table_task> on 2022-04-24 01:52:31.743537+00:00
[2022-04-24 01:52:59,331] {standard_task_runner.py:52} INFO - Started process 5154 to run task
[2022-04-24 01:52:59,396] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'local_2_gcs_2_bq_dag', 'crash_bigquery_external_table_task', 'manual__2022-04-24T01:52:31.743537+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/download_2_gcs_dag_V2.py', '--cfg-path', '/tmp/tmp1me6133v', '--error-file', '/tmp/tmp1g6q6tez']
[2022-04-24 01:52:59,398] {standard_task_runner.py:77} INFO - Job 46: Subtask crash_bigquery_external_table_task
[2022-04-24 01:52:59,737] {logging_mixin.py:109} INFO - Running <TaskInstance: local_2_gcs_2_bq_dag.crash_bigquery_external_table_task manual__2022-04-24T01:52:31.743537+00:00 [running]> on host 65283f82c270
[2022-04-24 01:53:00,020] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_2_gcs_2_bq_dag
AIRFLOW_CTX_TASK_ID=crash_bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-24T01:52:31.743537+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-24T01:52:31.743537+00:00
[2022-04-24 01:53:00,035] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-24 01:53:01,001] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=local_2_gcs_2_bq_dag, task_id=crash_bigquery_external_table_task, execution_date=20220424T015231, start_date=20220424T015259, end_date=20220424T015301
[2022-04-24 01:53:01,114] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-24 01:53:01,244] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
